src/core/shape_inference/include/fake_convert_shape_inference.hpp

```diff
@@ -13,16 +13,17 @@ template <class T, class TRShape = result_shape_t<T>>
 std::vector<TRShape> shape_infer(const FakeConvert* op, const std::vector<T>& input_shapes) {
     const auto inputs_count = input_shapes.size();
     const auto has_shifts_input = inputs_count == 3;
-    NODE_VALIDATION_CHECK(op, inputs_count  == 2 || has_shifts_input );
+    NODE_VALIDATION_CHECK(op, inputs_count == 2 || has_shifts_input);
     if (has_shifts_input) {
-        NODE_SHAPE_INFER_CHECK(op, input_shapes, input_shapes[1].compatible(input_shapes[2]),
-                        "FakeConvert scale shape is not compatible with shift shape.");
+        NODE_SHAPE_INFER_CHECK(op,
+                               input_shapes,
+                               input_shapes[1].compatible(input_shapes[2]),
```

Comment:
Just for future improvements. if at start of the function you make ref aliasies like
```
const& auto scales_shape = input_shape[1];
const& auto shifts_shape = input_shape[2];
```
then this check will look like `scales_shape.compatible(shifts_shapes)`.

and in exception message more or less there will be somthing like `... [data_shape],[scale_shape][shifts_shape ... scales_shape.compatible(shifts_shapes) ...` , so maybe explanation string is not required.

Reply:
Done aliases without changing string for now, will look into exception message if it could be simplified